### PR TITLE
README: Update it

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
Hi,

it seems that your commit message doesn't contain enough information. Don't worry, most people don't know about good commit messages when they join us, including very experienced developers :) Standards vary from project to project.

The most important part is the first line of your commit message - maintainers will use it to read through a large number of commits and thus it's important that it's short but most importantly concise. I'd heartily recommend reading http://coala.io/commit !

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix up the message for you - just tell us :).

Cheers!

**Comment by adtac** [comment]
Yes, sorry @Adrianzatreanu 

I'll fix it up immediately

File: README.rst [hunk 1]
```diff
@@ -1,23 +1,136 @@
...
-      ,d8,_    _,d8,   ,d8' d8,   ,d8b,,d8b,_ ,d8,   ,d8b,       Y           ,o8P
-    ooP""Y8888PP*"Y8888P"   "Y8888P"`Y88P'"Y88P"Y8888P"`Y8            oooo888P"
-
-.. Stop ignoring LineLengthBear
+.. image:: https://cloud.githubusercontent.com/assets/7521600/19196124/e6ea06dc-8cd1-11e6-84a8-c51c899a1f36.png
+
+coala-quickstart: personalized coala setup for your project
+-------------
+coala-quickstart is a tool that helps users to quickly get started
+with coala. It can generate a ``.coafile`` (coala's configuration
```
Comment (Adrianzatreanu) [inline README.rst]: i think a link to coala would be useful here if you click on "coala"

Comment (adtac) [inline README.rst]: website or repo?

Comment (Adrianzatreanu) [inline README.rst]: repo will lead to website so repo

**Comment by Adrianzatreanu** [comment]
ack 2dcb771

**Comment by adtac** [comment]
@rultor merge

**Comment by adtac** [comment]
reack https://github.com/coala/coala-quickstart/pull/37/commits/23cf288bfda852dc75c8998ad52ef933f517f780

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**